src/main/resources/application.properties

```diff
@@ -6,4 +6,5 @@ file.type-checksum=SHA-256
 file.tesseract-path=thirdParty/Tess4J/tessdata
 file.default-languageCode=eng
 file.default-language=english
+file.base-url=/api/v1.0/url
```

Comment:
why we have a property and constant with the same value? this property is never used

Reply:
It was removed